ocs_ci/ocs/node.py

```diff
@@ -267,3 +267,33 @@ def add_new_node_and_label_it(machineset_name):
     log.info(
         f"Successfully labeled {new_spun_node} with OCS storage label"
     )
+
+
+def get_ocs_worker_nodes():
```

Comment:
We already have functions to get the worker nodes get_typed_nodes, get_worker_nodes, can you check if you can use it?

Reply:
This 2 function get_typed_nodes, get_worker_nodes, will return only worker node not ocs worker node

Reply:
@prsurve please use https://github.com/red-hat-storage/ocs-ci/blob/f68241707339e2812b7ef6007dae8d72847553cf/ocs_ci/ocs/machine.py#L93